{{cookiecutter.repo_name}}/.github/actions/install-mdanalysis/action.yaml

```diff
@@ -0,0 +1,117 @@
+name: Install MDAnalysis
```

Comment:
This is getting migrated right?

Reply:
Yup -- into the other repo. I wanted to work through this while waiting for the CI on that one to get merged.